### PR TITLE
TestCSRCollected

### PR DESCRIPTION
**Pull request by psimovec**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1835090

Q: Is checking if CSR is collected enough? or we also need to check if the collected CSR is anonymized?

### CODE REVIEW

File: test/integration/bugs_test.go [hunk 1]
```diff
@@ -201,6 +202,37 @@ func TestArchiveContains(t *testing.T) {
...
+		ObjectMeta: metav1.ObjectMeta{Name: name},
+		Spec:       v1beta1.CertificateSigningRequestSpec{Request: certificateRequest},
+		Status:     v1beta1.CertificateSigningRequestStatus{},
+	})
+	e(t, err, "Failed creating certificate signing request")
+	defer func() {
+		clientset.CertificatesV1beta1().CertificateSigningRequests().Delete(name, &metav1.DeleteOptions{})
+		restartInsightsOperator(t)
+	}()
+	defer ChangeReportTimeInterval(t, 1)()
```
Comment (lina-nikiforova) [inline test/integration/bugs_test.go:229]: why do we need to change report time interval after the test? It wasn't changed during the test

Comment (psimovec) [reply test/integration/bugs_test.go:229]: ChangeReportTimeInterval returns callback function which reverts the changes. The report time interval is changed on this line, only callback function is defered. It's equivalent to:
`revertTimeInterval := ChangeReportTimeInterval(t, 1)`
`defer revertTimeInterval()`

Comment (lina-nikiforova) [reply test/integration/bugs_test.go:229]: ok, thanks!

**Comment by lina-nikiforova** [comment]
lgtm

**Pull request merged**